documentation/source/simulator_support.rst

```diff
@@ -256,6 +256,27 @@ Support is preliminary.
 Noteworthy is that despite GHDL being a VHDL simulator, it implements the VPI interface.
 
 
+.. _sim-ghdl-waveforms:
+
+Waveforms
+---------
+
+To get waveforms in VCD format, set the :make:var:`SIM_ARGS` option to ``--vcd=anyname.vcd``, 
```

Comment:
I think that a reference to https://ghdl.readthedocs.io/en/latest/using/Simulation.html#export-waveforms should be added. You can use `intersphinx` for that (see https://github.com/ghdl/ghdl-cosim/blob/master/doc/conf.py#L130-L134). I believe that using `intersphinx` helps with building a community. See also #1977.

Reply:
@umarcor I added the following.
The :make:var:`SIM_ARGS` option allows access to all [GHDL Command Line Run (-r) options](https://ghdl.readthedocs.io/en/latest/using/Simulation.html#using-simulation). This includes the generation of [other waveform formats](https://ghdl.readthedocs.io/en/latest/using/Simulation.html#export-waveforms) such as the compressed fst format and the ghw format which supports any VHDL type.

Reply:
LGTM. If you need to edit it again, I would uppercase FST and GHW. The former is *Fast Signal Trace* and the latter *GHDL Wave file*. See page 16 of http://gtkwave.sourceforge.net/gtkwave.pdf. Anyway, it's ok as-is. Thanks!